nep-24.mediawiki

```diff
@@ -0,0 +1,161 @@
+<pre>
+  NEP: 24
+  Title: NFT Royalty Standard
+  Author: Owen Zhang<zhangzhihong@ngd.neo.org>, Vincent Geneste<vincentgeneste@gmail.com>
+  Type: Standard
+  Status: Accepted
+  Created: 2022-09-05
+</pre>
+
```

Comment:
Table of contents is still not added. We need it exactly like in all NEPs, see https://github.com/neo-project/proposals/blob/master/nep-11.mediawiki#table-of-contents.

Reply:
@superboyiii can make the final format update

Reply:
> Table of contents is still not added. We need it exactly like in all NEPs, see https://github.com/neo-project/proposals/blob/master/nep-11.mediawiki#table-of-contents.

It's automatically generated. See: https://github.com/neo-project/proposals/blob/8a8b8c229262d46e553b00f9110eae3ccad6c1a3/nep-24.mediawiki
![c9ade7d951b311b9529c0c2b6a3c089](https://github.com/neo-project/proposals/assets/38493437/8ae920be-4b3d-417b-958a-d5597a34399f)

Reply:
Will add another link to C# demo, please don't merge.

Reply:
Got it, my bad.